src/Illuminate/Auth/Access/Gate.php

```diff
@@ -39,35 +39,35 @@ class Gate implements GateContract
     /**
      * All of the defined abilities.
      *
-     * @var array
+     * @var array<string, callable>
```

Comment:
And also here:
```suggestion
     * @var array<string, (callable(\Illuminate\Contracts\Auth\Authenticatable|null, ...mixed): )>
```

Reply:
abilities have a wider range of variations in terms of accepting arguments. See line 256.
This doc-block can be blamed to be incorrect later on.